packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -335,13 +335,19 @@ - (void)imagePickerController:(UIImagePickerController *)picker
 }
 
 - (void)imagePickerControllerDidCancel:(UIImagePickerController *)picker {
-  [_imagePickerController dismissViewControllerAnimated:YES completion:nil];
+  [self handleImagePickerControllerDismissed];
+}
+
+- (void)handleImagePickerControllerDismissed {
   if (!self.result) {
     return;
   }
   self.result(nil);
   self.result = nil;
   _arguments = nil;
+  // Dismiss image picker after cleaning up can precent race a condition,
```

Comment:
nit: typo (present) (also I'm not sure about the grammar, maybe s/can present race a condition/can lead to a race condition/)

Reply:
done